tests/main/snap-quota-memory/task.yaml

```diff
@@ -108,49 +108,53 @@ execute: |
   echo "And the slice is not active anymore"
   systemctl show --property=ActiveState "$sliceName" | MATCH "ActiveState=inactive"
 
-  echo "Creating a quota with a very small memory limit results in the service being unable to start"
-  snap set-quota too-small --memory=660KB go-example-webserver
-
-  # wait for systemd to finish trying to automatically restart it, we want 
-  # systemd to hit the start limit for this service, otherwise systemd 
-  # restarting the service will race with removing the quota directly below
-  # the unit will jump between SubState=running and SubState=auto-restart while
-  # trying to restart it in a loop, but after systemd has hit the limit it will
-  # give up and mark the unit as failed
-  retry --wait 1 -n 30 sh -c "systemctl show --property=SubState snap.go-example-webserver.webserver.service | MATCH 'SubState=failed'"
-
-  # clear "oom-killer" messages from dmesg or prepare-restore.sh will fail due
-  # to the oom-killer messages from go-example-webserver
-  tests.cleanup defer dmesg -c
-
-  echo "The systemd slice should be active"
-  sliceName="snap.$(systemd-escape --path too-small).slice"
-  systemctl show --property=ActiveState "$sliceName" | MATCH "ActiveState=active"
+  # snap.go-example-webserver.webserver.service is using about 540K and it is not killed
+  if not os.query is-fedora; then
```

Comment:
Wait, why not just reduce the memory limit to 448KB or something like that?

Reply:
I tried but the minimun is 660K iirc

Reply:
I think such low minimum doesn't work and the process forked from system may get killed even before it execs into the actual snap service.

Reply:
BTW. maybe we should have a larger limit and have the test program allocate and access enough memory at runtime to go over the limit but in a controlled way.